src/sonic-config-engine/minigraph.py

```diff
@@ -518,7 +624,7 @@ def parse_spine_chassis_fe(results, vni, lo_intfs, phyport_intfs, pc_intfs, pc_m
             if pc_member[0] == pc_intf:
                 intf_name = pc_member[1]
                 break 
-
+        
```

Comment:
Can you please remove spaces?

Reply:
Fixed in the latest commit

Reply:
i see multiple such instances here, suggest to change your editor setting to automatically remove the trailing spaces.